dist-material/release-docs/LICENSE

```diff
@@ -394,6 +399,8 @@ The following components are provided under the EPL License. See project link fo
 The text of each license is also included at licenses/LICENSE-[project].txt.
 
     logback 1.1.11: https://github.com/qos-ch/logback: EPL 1.0
+    logback-classic 1.2.3: https://github.com/qos-ch/logback: EPL 1.0
```

Comment:
Can you please also update the version in the previous line (401) 👆 , it should be 1.2.3 as well